rasterio/rio/clip.py

```diff
@@ -28,6 +29,11 @@
     default=True,
     help="Bounds in input's own projected coordinates (the default).")
 
+data_window_options = click.option(
+    '--clip-data-window',
```

Comment:
@mlinds what would you think about `--to-data-window` as the option name? The self-descriptiveness of `rio clip --to-data-window` would be nice.